packages/client/src/components/Navigation/Navigation.component.js

```diff
@@ -1,53 +1,65 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { NavLink, Link } from 'react-router-dom';
 import { useUserContext } from '../../userContext';
 import './Navigation.styles.css';
 import { Button } from '../Button/Button.component';
+import { FaBars } from 'react-icons/fa';
+import { ImCross } from 'react-icons/im';
 
 export const Navigation = () => {
   const { user, name, logout } = useUserContext();
+  const [showLinks, setShowLinks] = useState(false);
```

Comment:
Might make sense to rename this to `showMobileLinks` since it's only used for the mobile mode.

Reply:
Yeah ok 👍🏼